src/pages/AboutUs/styles.module.css

```
@@ -0,0 +1,173 @@
+.aboutUs {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 8%;
```

Eu pessoalmente acredito que fique mais adequado usar todas as medidas em viewport - `vh` para medidas verticais e `vw` para as horizontais (principalmente porque você já começou usando pra setar `width: 100vw` (mesma coisa para as outras medidas abaixo em porcentagem).

```suggestion
    margin-top: 25vh;
```